src/globus_sdk/services/auth/flow_managers/native_app.py

```diff
@@ -181,11 +184,10 @@ def get_authorize_url(self, query_params: dict[str, t.Any] | None = None) -> str
             "code_challenge": self.challenge,
             "code_challenge_method": "S256",
             "access_type": (self.refresh_tokens and "offline") or "online",
+            "prefill_named_grant": self.prefill_named_grant,
```

Comment:
I think this could follow our "standard" process:

```suggestion
            "prefill_named_grant": self.prefill_named_grant,
            **(query_params or {}),
```